### PR TITLE
Document exported functions and consts in pkg/mount

### PR DESCRIPTION
**Pull request by jamiehannaford**
Addresses #11594.

### CODE REVIEW

File: pkg/mount/mount.go [hunk 1]
```diff
@@ -38,9 +40,10 @@ func Mount(device, target, mType, options string) error {
 	return ForceMount(device, target, mType, options)
 }
 
-// Mount the specified options at the target path
-// reguardless if the target is mounted or not
-// Options must be specified as fstab style
+// Mount a filesystem according to the specified configuration, regardless if
```
Comment (LK4D4) [inline pkg/mount/mount.go]: What difference here from `Mount`?

File: pkg/mount/mountinfo.go [hunk 2]
```diff
@@ -1,7 +1,38 @@
 package mount
 
+// Reveals information about a particular mounted filesystem. This struct is
```
Comment (LK4D4) [inline pkg/mount/mountinfo.go]: Docstring should start from documenting entity name. For example:
```
MountInfo represents...
```
Also, that problem with other docstrings as well. You can find examples of docstrings in standard library.

File: pkg/mount/mount.go (hunk 1 above)
Comment (jamiehannaford) [inline pkg/mount/mount.go]: @LK4D4 AFAIK, `Mount` will check first to see if a similar mount exists. If it does, an error is returned; if it nothing exists, it calls `ForceMount`.

**Comment by jamiehannaford** [comment]
@LK4D4 Do you know why the Jenkins build is failing? It says:

> Error response from daemon: no such id: docker-pr4738

I've rebased my PR on `upstream/master`, so I'm not sure what the problem is.

**Comment by icecrime** [comment]
@jamiehannaford It's failing because your third commit in not signed. Could you please squash your commits into a single (signed) one? Thanks!

**Comment by jamiehannaford** [comment]
@icecrime @LK4D4 Ah, okay. I've squashed and signed them - so ready for review again :smile:

**Comment by icecrime** [comment]
Thanks! LGTM :+1:

**Comment by jfrazelle** [comment]
this is legit thanks!

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**